recipes/tardis/meta.yaml

```
@@ -0,0 +1,107 @@
+{% set name = "tardis-sn" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/tardis-sn/tardis.git
+  git_branch: master
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - setuptools
+    - python=3
```

If this isn't a `noarch` package the python version shouldn't be constrained as there will be a separate build for each version of Python.
```suggestion
    - python
```